assets/src/scripts/store/index.js

```diff
@@ -435,7 +459,7 @@ export const configureStore = function (initialState) {
                 median: false
             },
             currentDateRange: 'P7D',
-            requestedTimeRange: null,
+            customTimeRange: null,
```

Comment:
I was getting confused by requestedTimeRange and requestDT in the series part of the state. Changed this to customTimeRange because everything is requested and it seems more descriptive.